Testscripts/Linux/xfstesting.sh

```diff
@@ -182,7 +182,11 @@ Main() {
         if [ $FSTYP == "cifs" ]; then
             ConfigureCIFS "/root/test" "/root/scratch"
         else
-            ConfigureDisks "sdc" "sdc1" "sdc2" "$FSTYP" "test" "scratch"
+            testDisk=$(readlink -f /dev/disk/azure/scsi1/* | sed "s@/dev/@@g" | head -1)
```

Comment:
If the custom test parameter sets, it should be overridden whatever you implement here.

Reply:
Also a comment here with what you are trying to achieve and why is helpful

Reply:
@juhlee-microsoft this was never a custom test parameter, it was hard-coded and the reason this is necessary is because the linux next and other 5+krenels does not pick the attached storage drive as sdc it can have any drive letter

Reply:
scsi1 is documented somewhere? how are you testing this on non scsi devices?

Reply:
[Ref](https://docs.microsoft.com/en-us/azure/virtual-machines/troubleshooting/troubleshoot-device-names-problems)

```
$ tree /dev/disk/azure
/dev/disk/azure
├── resource -> ../../sdb
├── resource-part1 -> ../../sdb1
├── root -> ../../sda
├── root-part1 -> ../../sda1
└── scsi1
    ├── lun0 -> ../../../sdc
    ├── lun0-part1 -> ../../../sdc1
    ├── lun1 -> ../../../sdd
    ├── lun1-part1 -> ../../../sdd1
    ├── lun1-part2 -> ../../../sdd2
    └── lun1-part3 -> ../../../sdd3
```

Reply:
@stefanlupsa  As you removed the variable 'TEST_DEV=/dev/sdc1' out of files, my comment got void.